packages/lib/src/toolbar/controls/DomainWidget/BoundEditor.module.css

```diff
@@ -14,7 +14,7 @@
 }
 
 .value {
-  flex: 1 0 auto;
```

Comment:
We specify a `width` below, so the item will not shrink below that width regardless of `flex-shrink` and `flex-basis`.